uber/site_sections/preregistration.py

```diff
@@ -191,12 +191,31 @@ def banned(self, id):
         attendee, group = self._get_unsaved(id)
         return {'attendee': attendee}
 
+    def freereg_payment(self, session):
```

Comment:
Alternative function name: `free_prereg` or `process_free_prereg`. They're not paying for their free reg, so the current name isn't super clear about what's happening.